hstream/src/HStream/Server/Handler/Subscription.hs

```diff
@@ -494,7 +494,7 @@ sendRecords ctx@ServerContext {..} subState subCtx@SubscribeContext {..} = do
               else pure ()
               return False
             Right _ -> do
-              registerResend logId batchId shardRecordIds
+              -- registerResend logId batchId shardRecordIds
```

Comment:
Pls add some comments for why disabled? And should we need to re-enable in the future?